src/OpenTelemetry.AutoInstrumentation.BuildTasks/OpenTelemetry.AutoInstrumentation.BuildTasks.targets

```diff
@@ -30,7 +30,16 @@
       InstrumentationTargetItems="@(InstrumentationTarget)"
       RuntimeCopyLocalItems="@(RuntimeCopyLocalItems)"
       UseVerboseLog="$(UseVerboseLog)" />
-
   </Target>
 
+ <Target Name="VerifyIfRuntimeIdSpecified" BeforeTargets="Publish" Condition="'$(RuntimeIdentifier)' == '' AND $(DisableAutoInstrumentationCheckForRuntimeIdentifier) != 'true'">
+   <ItemGroup>
+    <Lines Include="RuntimeIdentifier (RID) is not set. Consider setting it to avoid copying native libraries for all of the platforms supported by the OpenTelemetry.AutoInstrumentation package." />
+    <Lines Include="See the docs at https://opentelemetry.io/docs/zero-code/net/nuget-packages/#using-the-nuget-packages for details." />
+    <Lines Include="In order to suppress this warning, set DisableAutoInstrumentationCheckForRuntimeIdentifier property to true." />
+   </ItemGroup>
+   <Warning Text="@(Lines, ' ')">
```

Comment:
This is the best workaround I've found to avoid very long line here and a clean warning message in dotnet publish results.